common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -70,6 +70,12 @@
         docClass += ' has-fixed';
     }
 
+    if (testCssSupportForPropertyAndValue('position', 'sticky')) {
```

Comment:
Minor point but because no browser that supports `CSS.supports` supports `sticky`, it might be worth just using supports instead of going into [all the else branch ](https://github.com/guardian/frontend/blob/d4422b4537165424e70a898d150db4e806ba04d6/common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js#L20)?